src/event/event.controller.ts

```diff
@@ -76,14 +79,61 @@ export class EventController {
 
   @Get(":id/attendees")
   async getAttendees(
-    @Param('id') id: string
```

Comment:
this makes sense to be part of the attendee module right? 
creating a controller in that module that makes contact with the service module